install_config/install/advanced_install.adoc

```diff
@@ -253,8 +253,8 @@ will be created within the
 xref:../../architecture/networking/sdn.adoc#architecture-additional-concepts-sdn[{product-title}
 SDN]. This network block should be private and must not conflict with any
 existing network blocks in your infrastructure to which pods, nodes, or the
-master may require access to, or the installation will fail. Defaults to
-`172.30.0.0/16`, and cannot be re-configured after deployment. If changing from the default, avoid `172.17.0.0/16`, which the *docker0* network bridge uses by default, or modify the *docker0* network.
+master may require access to, or the installation will fail. Reasonable value is
```

Comment:
I'm not sure I understand why this was changed to 'reasonable value is' ... is this not the default? And why is it reasonable?

Reply:
As https://github.com/openshift/openshift-ansible/pull/5430 indicates, it is no longer the default -- that's the reason behind this docs pull request.

I consider it reasonable because it works reasonably well, one reason being that these were the previous defaults.